src/common/shibuyaHelper.js

```diff
@@ -1,9 +1,13 @@
 import _ from 'lodash';
 import { ApiPromise, WsProvider } from '@polkadot/api';
+import { BN } from 'bn.js';
 
 import { getProxies, getProxyAccount } from './utils';
 import { Shibuya } from '../config';
 
+// frame_support::weights::constants::WEIGHT_PER_SECOND
```

Comment:
This is a good idea pointing it out 👍  Could you also link the line of code here?